Library/Formula/go.rb

```diff
@@ -64,9 +65,17 @@ def install
     end
 
     (buildpath/"pkg/obj").rmtree
-
     libexec.install Dir["*"]
     bin.install_symlink Dir["#{libexec}/bin/go*"]
+
+    if build.with? "godoc"
+      ENV.prepend_path "PATH", bin
+      ENV["GOPATH"] = buildpath/"brewtmp"
+
+      mkdir buildpath/"brewtmp"
+      system "go", "get", "golang.org/x/tools/cmd/godoc"
```

Comment:
Rather than fetching this via `go get`, I'd rather stage it as a resource and use the staged source to build.